src/crawlers/info.ts

```diff
@@ -412,3 +416,24 @@ export const _krmcmworldwidecom = () => {
     .textContent;
   return { name, brandKor, imageUrl, salePrice, originalPrice };
 };
+
+export const _www2hmcom = () => {
+  const name = document
+    .querySelector('meta[property="og:title"]')
+    .getAttribute('content')
+    .split('-')[0]
+    .trim();
+  const brandKor = 'hnm';
+  const imageUrl = document
+    .querySelector('meta[name="og:image"]')
+    .getAttribute('content');
+  const originalPrice = document
+    .querySelector('#product-price > div._2rpyBCXDllH8zsC70tsgt0 span')
+    .textContent.replace(/[^\d]+/g, '');
+
+  const salePrice = document
+    .querySelector('#product-price > div._2rpyBCXDllH8zsC70tsgt0 span')
+    .textContent.replace(/[^\d]+/g, '');
```

Comment:
```suggestion
  const originalPrice = document
    .querySelector('#product-price > div span')
    .textContent.replace(/[^\d]+/g, '');

  const salePrice = document
    .querySelector('#product-price > div span')
    .textContent.replace(/[^\d]+/g, '');
```
1. 해쉬값은 안 쓰는게 안전합니다! 변경의 여지가 많아서용
2. 세일하는 경우 대응이 안 돼있네요 ㅠㅠ

Reply:
47862b2